kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobActionDropdown.js

```diff
@@ -98,16 +98,24 @@ define([
                 return Promise.resolve(false);
             }
 
-            const jobIdList = jobManager.getJobIDsByStatus(
+            const jobList = jobManager.getCurrentJobsByStatus(
                 statusList,
                 Jobs.validStatusesForAction[action]
             );
-            if (!jobIdList || !jobIdList.length) {
+            if (!jobList || !jobList.length) {
                 return Promise.resolve(false);
             }
 
-            return JobMessages.showDialog({ action, statusList, jobIdList }).then((confirmed) => {
+            return JobMessages.showDialog({ action, statusList, jobList }).then((confirmed) => {
                 if (confirmed) {
+                    const jobIdList =
+                        action === 'retry'
+                            ? jobList.map((job) => {
+                                  return job.retry_parent || job.job_id;
+                              })
```

Comment:
for retrying, we should send the `retry_parent` ID if it exists, rather than the `job_id`